cmplibrary/src/test/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLibTest.java

```diff
@@ -103,23 +104,23 @@ public void run_followed_by_show_view(){
         lib.run();
         verify(timerMock).start();
         lib.showView(lib.webView);
-        verify(timerMock).cancel();
+        verify(timerMock , atLeast(1)).cancel();
```

Comment:
we should not make any assertions on the setup, not sure when it got here...

Reply:
@leodoin This is not setup method it is run_followed_by_show_view() test method.